gce/notebook/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update \
 USER $NB_USER
 
 RUN conda install --yes -c ioam -c bokeh/channel/dev \
-    bokeh=0.12.15dev1 \
+    bokeh=0.12.15dev3 \
```

Comment:
I suppose this change is what introduced the bug. @mrocklin, is there a known dask dashboard incompatibility with this version of bokeh?

Reply:
Yes.  I recommend updating distributed to latest release to resolve this